util/testutil/util.go

```diff
@@ -75,7 +78,7 @@ func StreamTestingLogToFile(destPath string) (func() error, error) {
 }
 
 // GetProjectRoot returns the path to the directory where the source code of this project reside.
-func GetProjectRoot(t *testing.T) string {
+func GetProjectRoot(t TestingFatal) string {
```

Comment:
Instead of having a new interface only so that error can be `Fatal`ed directly by `testing.T` and `log.Logger`, I'd prefer removing the arg and return (string, err) and let the caller errors out, regardless `testing.T` or `log.Logger`.

Reply:
same for `ApplyTextTemplate`

Reply:
Yea I think that makes things a little cleaner. I'll change it